### PR TITLE
bank_sql: switch to pgwire only

### PR DESCRIPTION
**Pull request by dt**
Currently panics the server (see https://github.com/cockroachdb/cockroach/issues/4691), so might want to want until that bug is fixed before landing this.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/examples-go/39)

### CODE REVIEW

File: bank/sql_bank/README.md [hunk 1]
```diff
@@ -33,5 +22,5 @@ Run against an existing cockroach node or cluster.
 # Launch your node or cluster in secure mode with certificates in [mycertsdir]
 # Find a reachable address:[mycockroach:26257].
 # Run the example with:
-./sql_bank https://mycockroach:26257/?certs=mycertsdir
+./sql_bank "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
```
Comment (tamird) [inline bank/sql_bank/README.md]: why the quotes?

Comment (dt) [inline bank/sql_bank/README.md]: `&` in the URL.

Comment (tamird) [inline bank/sql_bank/README.md]: Ah. Nit: use single quotes so it's clear there's no expansion going on.

Comment (dt) [inline bank/sql_bank/README.md]: disagree.

in fact, my actual command line _was_ using expansion for `~/.../root.client.key` and it isn't unreasonable to expect others to as well.

**Comment by dt** [comment]
Added casts to the placeholders to dodge the panic and fixed the extra arg, but discovered a bug in server's ROLLBACK responses. Should now be ready to merge once https://reviewable.io/reviews/cockroachdb/cockroach/4736 lands.

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/examples-go/39#-:-KBjI2d3oTkL-ufJSRPi:793035792)*

**Comment by dt** [comment]
k, all blockers resolved now so PTAL.

**Comment by petermattis** [comment]
LGTM

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/examples-go/39#-:-KBmKvdd0l03aBoBbfBo:676834233)*

**Pull request merged**